airbyte-integrations/connectors/source-alpha-vantage/source_alpha_vantage/spec.yaml

```diff
@@ -0,0 +1,53 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Alpha Vantage Spec
+  type: object
+  required:
+    - api_key
+    - symbol
+  properties:
+    api_key:
+      title: API Key
+      type: string
+      description: API Key
+      airbyte_secret: true
+      order: 0
+    symbol:
+      title: Symbol
+      type: string
+      description: Stock symbol (with exchange code)
+      examples:
+        - AAPL
+        - TSCO.LON
+      order: 1
+    interval:
+      title: Interval
+      type: string
+      description: |
+        Time-series data point interval. Required for intraday endpoints.
+      enum:
+        - 1min
+        - 5min
+        - 15min
+        - 30min
+        - 60min
+      default: 1min
+      order: 2
+    adjusted:
+      title: Adjusted?
+      type: boolean
+      description: |
+        Whether to return adjusted data. Only applicable to intraday endpoints.
+      default: false
+      order: 4
```

Comment:
This is a premium feature rigth?

Reply:
This only applies to the intraday endpoint. The `time_series_daily_adjusted` endpoint is premium, but this is mentioned in the docs.